### PR TITLE
Update to TileDB 2.13 [main-old branch]

### PR DESCRIPTION
**Pull request by johnkerl**
Core 2.13 aligns with TileDB-Py 0.19 and TileDB-R 0.17. Those are already released.

After this we can tag TileDB-SOMA 0.1.19.

Once TileDB Cloud is updated to 2.13 (REST server) then we can include TileDB-SOMA 0.1.19 in the UDF/notebook-image repos.

### CODE REVIEW

File: tools/ci/install-r-dependencies.sh
Comment (eddelbuettel) [inline tools/ci/install-r-dependencies.sh:17]: Nit level comment:  I would write/indent it more tightly (two lines max) and possibly also place it below the `remotes` install.  But most importantly it works.

File: apis/r/DESCRIPTION
Comment (eddelbuettel) [inline apis/r/DESCRIPTION:39]: I think we can remove the versioned dependency alltogether.

**Review by eddelbuettel — APPROVED** [review]
Looks fine to me apart from a stylistic nit or two.  So you may want to wait for a second approval before merging.

**Comment by johnkerl** [comment]
> Looks fine to me apart from a stylistic nit or two. So you may want to wait for a second approval before merging.

At this point this PR's sole job is to unbreak CI -- thanks for your review! :)

File: apis/r/DESCRIPTION
Comment (eddelbuettel) [inline apis/r/DESCRIPTION]: That is not right. We want tiledb, just without a version. So please make it

     tiledb,

**Review by eddelbuettel — CHANGES_REQUESTED** [review]

File: apis/r/DESCRIPTION
Comment (eddelbuettel) [reply apis/r/DESCRIPTION]: Unless `main-old` does not use `tiledb` at all?  I am rather surprised this survived the CI for R.

Comment (johnkerl) [reply apis/r/DESCRIPTION]: @eddelbuettel already fixed -- thanks!

**Review by eddelbuettel — APPROVED** [review]
LGTM

**Pull request merged**
